poetry.lock

```diff
@@ -1,60 +1,60 @@
 [[package]]
 name = "attrs"
-version = "21.2.0"
+version = "21.4.0"
```

Comment:
Worth noting I updated the whole poetry.lock which updated the versions of other dependencies along with ddtrace. Would this mean customers will have to update their attrs and other dependencies that ddlambda py relies on?

Reply:
Not sure if this is a problem or not

Reply:
Shouldn't be a problem, updating the dependencies with minor/patch updates is a good thing to do anyway.